ton_client/client/src/tvm/mod.rs

```diff
@@ -18,10 +18,12 @@ use ton_sdk::Contract;
 use crate::client::ClientContext;
 use crate::types::{ApiResult, ApiError};
 use crate::dispatch::DispatchTable;
+use ton_block::MsgAddressInt;
 
 #[derive(Serialize, Deserialize)]
 #[allow(non_snake_case)]
 pub(crate) struct ParamsOfLocalRunGet {
+    pub bocBase64: Option<String>,
```

Comment:
Maybe better pass struct `Contract` rather than separate fields `bocBase64`, `codeBase64`, `dataBase64`, `balance` and `last_paid`?

Reply:
The next release may be